gradle.properties

```diff
@@ -0,0 +1,8 @@
+# Build projects in parallel
+org.gradle.parallel=true
+
+# Enable configuration cache
+org.gradle.unsafe.configuration-cache=true
+
+# Enable build cache
+org.gradle.caching=true
```

Comment:
Vil dette gjøre byggene raskere?

Reply:
Alle disse flaggene gjorde byggene raskere lokalt (testet de hvert for seg også), men gjenstår å se hvor mye det hjelper på CI.
Vi har jo kraftigere maskiner enn det som kjører på CI, så er ikke sikkert `parallel` gjør like mye utslag der bl.a.